moped-editor/src/views/projects/newProjectView/SignalAutocomplete.js

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { TextField, CircularProgress } from "@material-ui/core";
+import { Autocomplete } from "@material-ui/lab";
+import { v4 as uuidv4 } from "uuid";
+
+const signalToFeatureCollection = signal => {
+  // Immitate a "drawn point" feature from a traffic signal feature
```

Comment:
if we change our approach to handling drawn points, this would be a dark dependency that could break.